README.md

```diff
@@ -319,6 +319,16 @@ ctx.run(q)
 // SELECT c.personId, c.phone FROM Person p, Contact c WHERE (p.age > 18) AND (c.personId = p.id)
 ```
 
+### concatMap
+```scala
+val q = quote {
```

Comment:
Maybe we could leave some comment regarding this operation from PR description?

Reply:
done